projects/plugins/jetpack/extensions/blocks/subscriptions/subscriptions.php

```diff
@@ -186,6 +202,25 @@ function register_newsletter_access_column( $columns ) {
 	);
 }
 
+/**
+ * Add a meta to prevent publication on firehose, ES AI or Reader
+ *
+ * @param int $post_id Post id being saved.
+ * @return void
+ */
+function maybe_add_paywalled_content_post_meta( int $post_id ) {
```

Comment:
Again, this function adds meta no matter what so maybe rename?

Reply:
Ah, true.